### PR TITLE
Framework for integration of consistent load vector due to point loads

### PR DESCRIPTION
**Pull request by kmokstad**
Currently for 2D LR-splines only (and non-mixed).
Not sure if this is the ideal solution, but at least it seem to work.

### CODE REVIEW

**Comment by kmokstad** [comment]
jenkins build this with downstreams ifem-elasticity=52 please

File: src/ASM/LR/ASMu2D.h [hunk 1]
```diff
@@ -310,6 +318,10 @@ class ASMu2D : public ASMunstruct, public ASM2D
   //! \return Local node number within the patch that matches the point, if any
   //! \return 0 if no node (control point) matches this point
   virtual int evalPoint(const double* xi, double* param, Vec3& X) const;
+  //! \brief Returns the element that contains a specified spatial point.
+  //! \param[in] param The parameters of the point in the knot-span domain
+  //! \return Local element number within the patch that containes the point
```
Comment (akva2) [inline src/ASM/LR/ASMu2D.h]: contains

**Comment by kmokstad** [comment]
changed to WIP, since we need this for tensor splines also

**Comment by kmokstad** [comment]
jenkins build this with downstreams ifem-elasticity=52 please

**Comment by kmokstad** [comment]
jenkins build this with downstreams ifem-elasticity=52 please

**Comment by kmokstad** [comment]
jenkins build this with downstreams ifem-elasticity=52 please

**Pull request merged**